docs/guide/util/common.md

```diff
@@ -1,9 +1,76 @@
-# Coming Soon
+# Common Helpers
 
-::: info
-This page is Under Construction 🏗️, whilst Moonwall has been built - the docs have not.
+## Moonwall Utils
 
-Please Stay tuned for more details of this feature!
-:::
+Moonwall Utils is a utils package for Moonwall that includes a suite of constants, functions, classes, and types that are useful when writing tests with Moonwall. 
 
-![underconstruction](/under-construction.png)
+### Installation
+
+Moonwall Utils can be installed with the following command:
+
+```bash
+npm i @moonwall/util
+```
+
+### Moonwall Utils Reference
+
+#### Generic Account Constants
+
+Moonwall Utils has a variety of account constants intended for generic substrate networks. A full list of these constants is available in `accounts.js` of the `constants` folder of Moonwall Utils.
```

Comment:
Not sure this line makes sense.  Also it's worth linking to the actual file in github.